### PR TITLE
refactor(frontend): Move telemetry to database

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Basicly our telemetry data was stored on a disk and now we want to move those data to gcp database

### CODE REVIEW

File: backend/src/db-utils.js [hunk 1]
```diff
@@ -101,16 +104,18 @@ const queryNodeValidators = async () => {
...
-        `,
-  ]);
+  return await queryRows(
+    [
+      `SELECT ip_address AS ipAddress, account_id AS accountId, node_id AS nodeId,
+        last_seen AS lastSeen, last_height AS lastHeight,status,
+        agent_name AS agentName, agent_version AS agentVersion, agent_build AS agentBuild,
+        latitude, longitude, city
+      FROM nodes
+      WHERE last_seen > (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('seconds', NOW() - INTERVAL '60 seconds')) AS bigint) * 1000)
```
Comment (shelegdmitriy) [inline backend/src/db-utils.js:114]: @frol  I'm not sure about this condition

**Review by frol — CHANGES_REQUESTED** [review]
Overall, looks good to me. Please, address the comments

File: backend/src/db-utils.js (hunk 1 above)
Comment (shelegdmitriy) [reply backend/src/db-utils.js]: I've fixed this. It seems there is no need to update `saveNodeIntoDatabase `

File: backend/src/db-utils.js [hunk 2]
```diff
@@ -21,8 +22,10 @@ function getSequelize(dataSource) {
       return models.sequelizeIndexerBackendReadOnly;
     case DS_ANALYTICS_BACKEND:
       return models.sequelizeAnalyticsBackendReadOnly;
+    case DS_TELEMETRY_BACKEND:
+      return models.sequelizeTelemetryBackendReadOnly;
     default:
-      return models.sequelizeLegacySyncBackendReadOnly;
+      return models.sequelizeTelemetryBackendReadOnly;
```
Comment (frol) [inline backend/src/db-utils.js:28]: Let's throw an exception here

**Review by frol — APPROVED** [review]

**Pull request merged**